doc/schema/schema.md

```diff
@@ -95,7 +95,7 @@ Parameters of the sampling algorithm and sampling backend to be used for analysi
 
 ### Out of sample `posterior_predictive` samples
 #### `predictions`
-Out of sample posterior predictive samples p(y'|y). Sample should match `posterior` samples. Its variables should have a counterpart in `posterior_predictive`. However, variables in `predictions` and their counterpart in `posterior_predictive` may not share coordinates.
+Out of sample posterior predictive samples p(y'|y). Samples should match `posterior samples. Its variables should have a counterpart in `posterior_predictive`. However, variables in `predictions` and their counterpart in `posterior_predictive` can have different coordinates.
 
 #### `predictions_constant_data`
 Model constants used to get the `predictions` samples. Its variables should have a counterpart in `constant_data`. However, variables in `predictions_constant_data` and their counterpart in `constant_data` may not share coordinates.
```

Comment:
Here again, "may not share coordinates" is not the same as "possibly do not share coordinates":  "may not" is the same as "are forbidden to."  So I think that you should change that to "may have different coordinates."

As before, I think it would be helpful to distinguish between having different coordinates and having different coordinate values.  Or possibly "different coordinate definitions."  The xarray terminology is not helpful here.